code/modules/tgui/tgui.dm

```diff
@@ -275,7 +275,10 @@
 		return FALSE
 	switch(type)
 		if("ready")
-			initialized = TRUE
+			if(!initialized)
+				initialized = TRUE
+			else // user refreshed the window
+				send_full_update(null, TRUE)
```

Comment:
this fixes an issue where if you refreshed the window on the client then all static data would go lost